angular.json

```diff
@@ -129,6 +129,7 @@
   "cli": {
     "schematicCollections": [
       "@angular-eslint/schematics"
-    ]
+    ],
+    "analytics": false
```

Comment:
I did not explicitly add this, not sure which npm, nvm, node, or ng command I ran did this but I can revert it.